packages/block-editor/src/components/block-navigation/tree.js

```diff
@@ -3,29 +3,59 @@
  */
 
 import { __experimentalTreeGrid as TreeGrid } from '@wordpress/components';
-import { useEffect, useMemo, useRef } from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
+import { useCallback, useEffect, useMemo, useRef } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
 import BlockNavigationBranch from './branch';
 import { BlockNavigationContext } from './context';
+import useBlockNavigationClientIds from './use-block-navigation-client-ids';
 import useBlockNavigationDropZone from './use-block-navigation-drop-zone';
+import { store as blockEditorStore } from '../../store';
+
+const noop = () => {};
 
 /**
  * Wrap `BlockNavigationRows` with `TreeGrid`. BlockNavigationRows is a
  * recursive component (it renders itself), so this ensures TreeGrid is only
  * present at the very top of the navigation grid.
  *
- * @param {Object}  props                                          Components props.
- * @param {boolean} props.__experimentalFeatures                   Flag to enable experimental features.
- * @param {boolean} props.__experimentalPersistentListViewFeatures Flag to enable features for the Persistent List View experiment.
+ * @param {Object}   props                                          Components props.
+ * @param {Array}    props.blocks                                   Custom subset of block client IDs to be used instead of the default hierarchy.
+ * @param {Function} props.onSelect                                 Block selection callback.
+ * @param {Array}    props.selectedBlockClientIds                   The client IDs of the (single or multi-) selected blocks.
+ * @param {boolean}  props.showNestedBlocks                         Flag to enable displaying nested blocks.
```

Comment:
Even though we almost have this enabled everywhere, personally I'd still prefer to `opt-in` rather than `opt-out`.

Reply:
I don't have a strong opinion either way, to be honest.
I haven't paid much attention to that prop; it has been there for quite a while now, and I have no idea why it was introduced instead of just defaulting it to true. 🤔